### PR TITLE
inetutils: update to 2.0.

### PR DESCRIPTION
**Pull request by glaulher**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by Chocimier** [comment]
checkdepends="iana-etc net-tools" fixes tests, but some are stil skipped. Reset revision to 1.

File: srcpkgs/inetutils/template [hunk 1]
```diff
@@ -19,6 +19,11 @@ subpackages="inetutils-dnsdomainname inetutils-ftp inetutils-hostname
 
 CFLAGS="-fcommon"
 
+# broken tests
+checkdepends="iana-etc net-tools"
```
Comment (ericonr) [inline srcpkgs/inetutils/template:23]: Please include this line under `makedepends`.

File: srcpkgs/inetutils/template [hunk 2]
```diff
@@ -19,6 +19,11 @@ subpackages="inetutils-dnsdomainname inetutils-ftp inetutils-hostname
...
+# tests can't run in CI
+make_check=extended
```
Comment (ericonr) [inline srcpkgs/inetutils/template:25]: Even with the added check dependencies? :/

Comment (glaulher) [reply srcpkgs/inetutils/template:25]: even with the added check depedencias  the tests broke.
I will put it below the makedepends.

**Pull request merged**